src/slice.rs

```diff
@@ -88,21 +88,21 @@ impl<T> Debug for Slice<T> {
     }
 }
 
-pub const fn from_str(s: &str) -> Slice<&str> {
+pub fn from_str(s: &str) -> Slice<&str> {
     Slice::wrap(
         FLSlice {
             buf: s.as_ptr().cast::<c_void>(),
-            size: s.len() as u64,
+            size: s.len().try_into().unwrap(),
```

Comment:
Are we confortable with unwrapping here?

Reply:
Currently the raw cast make it fail to compile on 32-bits arch because:
(64bits) usize => u64 => u64
(32bits) usize => u64 => u32
Using try_into() makes us getting right inference, and into() - u32/u64 is not available because no guarantee every arch can support it. However in our case we only target 32 & 64 bits platforms.

Reply:
Have you tried `.size_t_is_usize(true)` when generating bindings?